### PR TITLE
First two homework assignments from PyLec_2

### PR DESCRIPTION
**Pull request by IlyaOrlov**

### CODE REVIEW

File: Practice/kruglov/pep8task.py [hunk 1]
```diff
@@ -0,0 +1,90 @@
...
+from time import *
+
+# Class names must begin with a capital letter
+class shuffler:
+
+    def __init__(self):
+        self.map = {}
+
+    def rename(self, dirname, output):
+        # Vertical alignment required
```
Comment (IlyaOrlov) [inline Practice/kruglov/pep8task.py]: Here and below... alignment with what?

File: Practice/kruglov/pep8task.py [hunk 2]
```diff
@@ -0,0 +1,90 @@
...
+          mp3s = []
+        for root, directories, files in os.walk(dirname):
+            for file in files:
+                if file[-3:] == '.mp3':
+                    mp3s.append([root, file])
+        for path, mp3 in mp3s:
+            # The variable name matches the system
```
Comment (IlyaOrlov) [inline Practice/kruglov/pep8task.py]: What variable name?

File: Practice/kruglov/pep8task.py [hunk 3]
```diff
@@ -0,0 +1,90 @@
...
+            hashname = self.generateName() + '.mp3'
+            # The variable name matches the system
+            self.map[hashname] = mp3
+            # Two extra brackets
```
Comment (IlyaOrlov) [inline Practice/kruglov/pep8task.py]: Which of 4 brackets?

File: Practice/kruglov/pep8task.py [hunk 4]
```diff
@@ -0,0 +1,90 @@
...
+          mp3s = []
+        for root, directories, files in os.walk(dirname):
+            for file in files:
+               # Vertical alignment required
+               if file[-3:] == '.mp3':
+                    mp3s.append({root, file})
+        # The variable name matches the system
+        for path, hashname in mp3s:
+            # The variable name matches the system
+            # One extra brackets
```
Comment (IlyaOrlov) [inline Practice/kruglov/pep8task.py]: Which bracket is extra?

File: Practice/kruglov/home_task_1/First.py [hunk 5]
```diff
@@ -0,0 +1,18 @@
+# 1. Function returns the maximum number of two variables
```
Comment (IlyaOrlov) [inline Practice/kruglov/home_task_1/First.py]: I guess this comment is related to the 2nd function and vice versa.

File: Practice/kruglov/pep8task.py [hunk 6]
```diff
@@ -0,0 +1,91 @@
...
+
+    def rename(self, dirname, output):
+        # Two extra spaces
+          mp3s = []
+        for root, directories, files in os.walk(dirname):
+            for file in files:
+                if file[-3:] == '.mp3':
+                    mp3s.append([root, file])
+        for path, mp3 in mp3s:
+            # Failed variable name "hashname"
```
Comment (IlyaOrlov) [inline Practice/kruglov/pep8task.py]: Why is it failed? What style rule does this "hashname" contradict? PyCharm may highlight probable spelling mistakes, but it's not a style issue.

File: Practice/kruglov/pep8task.py [hunk 7]
```diff
@@ -0,0 +1,91 @@
...
+            hashname = self.generateName() + '.mp3'
+            # Failed variable name "hashname"
```
Comment (IlyaOrlov) [inline Practice/kruglov/pep8task.py]: The same as above.

File: Practice/kruglov/pep8task.py (hunk 6 above)
Comment (Darshniderd) [reply Practice/kruglov/pep8task.py]: Ok i understand. Fixed that.

File: Practice/kruglov/pep8task.py (hunk 7 above)
Comment (Darshniderd) [reply Practice/kruglov/pep8task.py]: The same as above)

**Pull request merged**